assert/assertions.go

```diff
@@ -202,6 +213,68 @@ func isNumericType(t reflect.Type) bool {
 	return t.Kind() >= reflect.Int && t.Kind() <= reflect.Complex128
 }
 
+// listsAreEqualValues gets whether two lists(arrays, slices) are equal, or if their
```

Comment:
It doesn't get if lists are equal because lists aren't comparable.

```suggestion
// listsAreEqualValues returns true if the expected and actual lists (arrays or slices) are the same length, and each index in both lists is convertible to the larger type and equal.
```